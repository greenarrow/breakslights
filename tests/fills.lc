MR003
MA003
A000 F00ff00 B000000 IF S001 P001 D255
A001 F00ff00 B000000 IO S001 P001 D255 L006
A002 F00ff00 B000000 IR S001 P001 D255 L006
R000 N000
R001 N001
R002 N002
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
