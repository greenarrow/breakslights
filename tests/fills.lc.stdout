00 00ff00 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000
01 000000 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000
02 000000 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000
00 00ff00 00ff00 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000
01 000000 000000 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000
02 000000 000000 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000
00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000 000000 000000 000000 000000
01 000000 000000 000000 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000
02 000000 000000 000000 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000
00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000 000000 000000 000000
01 000000 000000 000000 000000 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000
02 000000 000000 000000 000000 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000
00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000 000000 000000
01 000000 000000 000000 000000 000000 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000
02 000000 000000 000000 000000 000000 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000
00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000 000000
01 000000 000000 000000 000000 000000 000000 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00
02 000000 000000 000000 000000 000000 000000 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00
00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000
01 00ff00 000000 000000 000000 000000 000000 000000 00ff00 00ff00 00ff00 00ff00 00ff00
02 00ff00 000000 000000 000000 000000 000000 000000 00ff00 00ff00 00ff00 00ff00 00ff00
00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000
01 00ff00 00ff00 000000 000000 000000 000000 000000 000000 00ff00 00ff00 00ff00 00ff00
02 00ff00 00ff00 000000 000000 000000 000000 000000 000000 00ff00 00ff00 00ff00 00ff00
00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000
01 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000 000000 00ff00 00ff00 00ff00
02 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000 000000 00ff00 00ff00 00ff00
00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000
01 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000 000000 00ff00 00ff00
02 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000 000000 00ff00 00ff00
00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000
01 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000 000000 00ff00
02 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000 000000 00ff00
00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00
01 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000 000000
02 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000 000000
00 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000
01 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000 000000
02 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000 000000
00 00ff00 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000
01 000000 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000
02 000000 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000
